server.js

```diff
@@ -267,33 +267,41 @@ app.post('/slambook', async (req, res) => {
 //   }
 // });
 
-// app.put('/slambook/:id', async (req, res) => {
-//   try {
```

Comment:
Consider returning a more specific response message, such as the updated entry itself, instead of a generic 'Entry Updated' message.